implementer-guide/user-management/guest-user.md

```diff
@@ -1,37 +1,41 @@
 ---
 title: Guest user
 page_title: Guest user
-description: Guest user
+description: Explains the Guest user feature and its purpose and limitations
 slug: guest-user
-tags: users,guest
+tags: users,guest,share,embed
 published: True
 position: 400
 ---
 
 # Guest user
+The Guest user is a built-in user account That is created upon the initial setup of the Telerik Report Server. The Guest's purpose is to be used for report document preview and sharing, either using a sharing link to a report, a report viewer control embedded in a client application, or the reports API.
 
-Guest user is a special built-in user which is created upon the initial setup of Telerik Report Server. The Guest's purpose is to be used for read-only connections to report server, either using the reports API or sharing a link to a report.
+## Guest User Administration
+The Guest user is not enabled by default. To enable it, navigate to the _User Management -> Users_ view, select the Guest user from the list of users and turn on the _Enabled_ switch.
```

Comment:
reading about the Enabled switch leaves a sense of mismatching UI - I would expect to see something like a switch (toggle) control, but I see a simple checkbox instead.
"... check the Enabled checkbox" or "tick the Enabled checkbox (to avoid repetition)" would do fine, IMHO.

Reply:
Better. Applied. Thanks!